src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,18 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => <>Put the comment here</>;
+export const CommentInfo: React.FC<{ comment: Comment }> = ({ comment }) => (
```

Comment:
Just to make it cleaner, let's extract values from comment here or at the very first line of the component: 

`CommentInfo: React.FC<{ comment: Comment }> = ({ comment: { name,... } }) => (`
||
```
export const CommentInfo: React.FC<{ comment: Comment }> = ({ comment }) => (
    const { name, ... } = comment; 
```